src/GenreSelect/GenreSelect.js

```diff
@@ -0,0 +1,26 @@
+import react from "react";
+
+function GenreSelect(props) {
+  const Genres = ['Action', 'Comedy', 'Drama', 'Fantasy']
+
+  return (<>
+    <nav className="navbar navbar-expand-lg navbar-light bg-light">
+      <span className="navbar-brand">Genres</span>
+      <button className="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
why do we need all these attributes? where did you find them? :)

Reply:
changed to createElement with small peace of code)